source/_integrations/neato.markdown

```diff
@@ -53,6 +53,13 @@ neato:
 4. Add Neato Botvac via *Configuration* -> *Integrations*
 5. Follow the instructions. After that, all the entities will automatically show up in Home Assistant.
 
+<div class='note'>
+
+Note that if you have multiple robots registered to your Neato account, make sure that they are all connected, or you might get an error when adding the integration. Delete any old or unused robots from your Neato account before setting up the integration to avoid this.
+
+</div>
```

Comment:
This is a bug, that should be fixed, not documented. Have you filed an issue in the core repository?

Reply:
Core issue added:

https://github.com/home-assistant/core/issues/45241

Reply:
Alright, closing up the PR from this end 👍